FingerBlox/app/src/main/java/org/fingerblox/fingerblox/ImageProcessing.java

```diff
@@ -721,12 +725,15 @@ private double calculateMean(double[] m) {
      *
      * @return
      */
-    private Mat snapShotMask(int rows, int cols, int offset) {
+    private Mat snapShotMask(int rows, int cols, int padding) {
+        /*
+        Some magic numbers. We have no idea where these come from?!
         int maskWidth = 260;
         int maskHeight = 160;
+        */
 
         Point center = new Point(cols / 2, rows / 2);
-        Size axes = new Size(maskWidth - offset, maskHeight - offset);
+        Size axes = new Size(cols/2 - padding, rows/2 - padding);
```

Comment:
The `/2` is for correcting the downscaling right? Wouldn't it be better to make that a class constant?